### ISSUE TITLE
Confusing error message if OneDrive (or other cloud service) session expired

### ISSUE THREAD

**Issue by overheadhunter** [cryptomator/ios#242]
### Please agree to the following

- [X] I have searched [existing issues](https://github.com/cryptomator/ios/issues?q=) for duplicates
- [X] I agree to follow this project's [Code of Conduct](https://github.com/cryptomator/ios/blob/develop/.github/CODE_OF_CONDUCT.md)

### Summary

When trying to unlock a vault with FaceID, I got an error message saying "unlock cancelled" when the app was really unable to renew the session with the cloud storage provider.

### System Setup

- iOS: 15.5 (19F77)
- Cryptomator: 2.3.0 (967)

### Cloud Type

OneDrive

### Steps to Reproduce

1. Add a vault on OneDrive (other services may behave the same and have even shorter living sessions?)
2. Don't access the vault for several weeks (according to my logs, > 4-6 weeks?)
3. Try to access the vault after the session timed out

### Expected Behavior

1. Ideally, Cryptomator can refresh the session itself
2. Alternatively, it should show a meaningful error AND allow to manually re-login to existing accounts (as you probably don't want to logout and remove corresponding vaults)

### Actual Behavior

Unlock fails with generic error message (can't recall the exact text).

Furthermore, there is no obvious way to renew the session. Even in the main app under cloud services, the account doesn't show up as expired.

### Reproducibility

Intermittent

### Relevant Log Output

```shell
# file provider:
2022/07/14 19:25:01:873  enumerateItems getAdapter failed with: biometricalUnlockCanceled(biometryName: "Face ID") for identifier: NSFileProviderItemIdentifier(_rawValue: NSFileProviderRootContainerItemIdentifier)

# main app:
2022/07/14 19:26:10:787  Error: unauthorized
```

### Anything else?

**Workaround:** No need to log out and in again on OneDrive. Instead, just add the same account a second time. The app is able to merge this. This way, you can keep the vaults.

**Comment by tobihagemann** [comment]
Depends on #18.

**Comment by tobihagemann** [comment]
Confusing error message fixed in 2.6.3. However, we're still not quite done yet, see #387. Even though this issue is not _fully_ fixed, I'll close this in favor of #387.
